package.json

```diff
@@ -127,9 +127,9 @@
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "page-title-service": "drugis/page-title-service#1.0.5",
     "raw-loader": "~1.0.0",
-    "react": "^16.13.1",
+    "react": "^17.0.1",
```

Comment:
sneaking a potentially big upgrade in there; what's new?

Reply:
crap thought I changed it back ':) Wanted to try not having to type 'import React from 'react'' in every file, but couldn't get it to work. Shall I change it back or just leave it in?

Reply:
if it breaks nothing I don't mind, just need to be careful when the front number goes up :)